modules/blog/src/main/DailyFeed.scala

```diff
@@ -85,8 +98,10 @@ final class DailyFeed(coll: Coll, cacheApi: CacheApi)(using Executor):
           ),
         "content" -> nonEmptyText(maxLength = 20_000).into[Markdown],
         "public"  -> boolean
-      )(Update.apply)(unapply)
-    from.fold(form)(form.fill)
+      )((Update.apply))(Update.formUnapply)
+    from.fold(form):
+      cache.clear() // we need lastRev to be correct
```

Comment:
uh, but this only instanciates a form object? It does not write anything to DB

Reply:
the next line after cache.clear() is form.fill(_)

Reply:
we should be clearing the cache when the content changes. On write, not on read